document/value.go

```diff
@@ -250,15 +250,10 @@ func (v Value) MarshalJSON() ([]byte, error) {
 		return strconv.AppendInt(nil, v.V.(int64), 10), nil
 	case DoubleValue:
 		f := v.V.(float64)
-		abs := math.Abs(f)
-		fmt := byte('f')
-		if abs != 0 {
-			if abs < 1e-6 || abs >= 1e21 {
-				fmt = 'e'
-			}
+		if float64(f) == float64(int(f)) {
```

Comment:
f is already a float64, I think we avoid casting it here and below for clarity

Reply:
Done